alembic.ini

```diff
@@ -2,8 +2,15 @@
 
 [alembic]
 # path to migration scripts
+# WARN: This needs to be changed before "alembic" calls depending on
+# which database we are revisioning, the RUN or the PRODUCT CONFIG one.
+
+# Value for RUN databases:
```

Comment:
We can either have separate directories with separate alembic.ini, or we can share the ini file:

```
f you really wanted two completely independent sets of migration scripts, then you'd run two migration environments.  

They can share the same alembic.ini like this:

[my_db_one]
sqlalchemy.url =

[my_db_two]
sqlalchemy.url =

you then run alembic with "alembic -n my_db_one" or "alembic -n my_db_two".    The "default" config area is set by -n.

A single env.py script can get multiple database URLs in any way it wants, as it determines how config is accessed.   If you look in the multidb/env.py script, you'll see it's pulling multiple database urls from one section using config.get_section(name) - config file:

[alembic]
# path to migration scripts
script_location = ${script_location}

# template used to generate migration files
# file_template = %%(rev)s_%%(slug)s

databases = engine1, engine2

[engine1]
sqlalchemy.url = driver://user:pass@localhost/dbname

[engine2]
sqlalchemy.url = driver://user:pass@localhost/dbname2

usage:

    config = context.config

    db_names = config.get_main_option('databases')

    for name in re.split(r',\s*', db_names):
        engines[name] = rec = {}
        rec['engine'] = engine_from_config(
                                    config.get_section(name),
                                    prefix='sqlalchemy.',
                                    poolclass=pool.NullPool)

Over here I have both forms of multi db at the same time.   There's two migration environments, and one migration environment does two databases that are largely mirrored, so three databases total.   All three make use of a common env.py script that's in my application as a library, they then implement an env.py in the migration environment that draws upon the myapp/lib/env.py script for common features.

You can pass instructions to a single env.py that may be controlling multiple databases using --tag:

"alembic --tag my_tag"

"my_tag" is available in env.py as context.get_tag_argument().   You can use that to conditionally run migrations on one database or the other.

This is all DIY.  Multi-database migrations can happen in many different ways so you'd need to build the approach that suits your situation best.
```